tests/gatsby-starter-default/gatsby-config.js

```diff
@@ -12,7 +12,7 @@ module.exports = {
       resolve: 'gatsby-source-plone',
       options: {
         baseUrl: 'http://localhost:8080/Plone/',
-        token: process.env.TOKEN,
+        token:  process.env.TOKEN,
```

Comment:
Please, remove extra space. There should be no change here.

Reply:
I am very sorry. I am making such a silly mistake. I should check it before asking for a review.